Quadriga/src/main/java/edu/asu/spring/quadriga/rest/SearchNetworkRestController.java

```diff
@@ -108,10 +113,19 @@
             return new ResponseEntity<String>(errorMsg, HttpStatus.NOT_FOUND);
         }
 
+        // Fetch ConceptPower entries related to the conceptId
+        ConceptpowerReply reply = conceptpowerConnector.getById(conceptId);
+        List<String> alternativeIdsForConcept = null;
+        if (reply != null && reply.getConceptEntry().size() > 0) {
+            alternativeIdsForConcept = reply.getConceptEntry().get(0).getAlternativeIdList();
+        }
+        if(alternativeIdsForConcept == null){
+            alternativeIdsForConcept = new ArrayList<String>();
+        }  
```

Comment:
there isn't really a reason to have this code here in the controller, instead of in the class that is actually using the alternative ids. Just creates duplicate code (with the other controller).